### ISSUE TITLE
Treeview empty / unable to view items in 2.1.7

### ISSUE THREAD

**Issue by syphernl** [nilsteampassnet/TeamPass#83]
With the upgrade to 2.1.7 the tree is empty and shows no items.
On home, it shows the last created items but clicking them leads to an error:

```
You are not allowed to see that Item!
Author: undefined
Restricted to: undefined
```

Looks the output given by `sources/items.queries.php` in 2.1.7 is just 1 line (probably an error that gets encoded?) while in 2.1.5 its a big output spanning multiple lines.

**Comment by isco1988** [comment]
i don't see icone home in the version 2.1.27.36 why ?

**Comment by mrivero** [comment]
> I'am having this issue.
> The problem is that not inserted ('at_creation') record into log_items table at creation.
> Is there any fix?

I'm migrating a server to a new one (old one is Centos, new one is Ubuntu) but similar Mysql and php versions. So I exported (backed up) the mysql DB, imported it in the new server, and then copied all the Teampass files (using 2.1.19 still) and everything works great EXCEPT adding a new item which gives the "you are not allowed to see that Item! Author: undefined". 
Doing some research I stumbled upon this very page, and I can confirm that if I manually add an entry in "teampass_log_items" with the "at_creation" value on the "action" column for that id_item, then the newly added item is visible again in the website.

Is there any fix, or where I can look into modifying for this to happen when the item gets added??

Thanks a lot, hoping that someone still reads this page!

**Comment by mrivero** [comment]
Regarding the above question, I kept testing and found the issue and fixed it. If this helps anyone: in the "items.queries.php" file, there was a "queryInsert" into the "log_items" but that insert didn't have all the fields in there, "raison" was missing. Probably in the older version of mysql that gets inserted anyway with an empty value, but I had to manually add that field into the query.
Anyway, very specific to this very old version of Teampass (2.1.9) and probably the different mysql versions, but I'm glad it's working again!!!
